### PR TITLE
Add fit long words feature to tropic paragraph

### PR DESCRIPTION
**Pull request by chekaninmaxim**

### CODE REVIEW

**Comment by chekaninmaxim** [comment]
[Add fit long words feature for tropic wigify](https://trello.com/c/E5bYyKYE/849-add-fit-long-words-feature-for-tropic-wigify)

File: lib/tropic/tropic_paragraph.flow [hunk 1]
```diff
@@ -651,9 +662,12 @@ TParaLine(words : [TParaWord], indent : double);
 // To remember changed in reflowTParaWords words and thier indexes
 WordsAndIndexes(words : [TParaWord], indexes : [int]);
 // Return type for reflowTParaWordsRec
-LinesAndWords(lines : [TParaLine], words : WordsAndIndexes);
+TReflowResult ::= TOverflowCase, TReflowedLinesAndChanges;
```
Comment (ykuzmenko73) [inline lib/tropic/tropic_paragraph.flow:665]: TReflowResult should be TReflowResultAcc

File: lib/tropic/tropic_paragraph.flow [hunk 2]
```diff
@@ -651,9 +662,12 @@ TParaLine(words : [TParaWord], indent : double);
...
+	TOverflowCase(width: double);
+	TReflowedLinesAndChanges(lines : [TParaLine], changes : WordsAndIndexes);
+	initialLines = TReflowedLinesAndChanges([], WordsAndIndexes([], []));
 // Return type for reflowTParaWords
-metricsAndWords(metrics : Pair<WidthHeight, double>, words : WordsAndIndexes);
+MetricsChangesAndScale(metrics : Pair<WidthHeight, double>, words : WordsAndIndexes, scaleM : Maybe<double>);
```
Comment (ykuzmenko73) [inline lib/tropic/tropic_paragraph.flow:670]: MetricsChangesAndScale should be TReflowResult

**Review by ykuzmenko73 — COMMENTED** [review]
LGTM, small renaming

**Pull request merged**